kafka_consumer/datadog_checks/kafka_consumer/client/confluent_kafka_client.py

```diff
@@ -14,20 +19,20 @@ def __init__(self, config, tls_context, log) -> None:
         self._highwater_offsets = {}
         self._consumer_offsets = {}
         self._tls_context = tls_context
-    
+
     @property
     def kafka_client(self):
         if self._kafka_client is None:
             # self.conf is just the config options from librdkafka
-            self._kafka_client = AdminClient(self.conf)
+            self._kafka_client = AdminClient({"bootstrap.servers": self.config._kafka_connect_str})
```

Comment:
Note for another PR: once we will be done with the refactor, we might update the config object to avoid having protected attributes (e.g `kafka_connect_str` instead of `_kafka_connect_str`)